src/draw/lv_draw_vector.c

```diff
@@ -14,6 +14,23 @@
 #include "../stdlib/lv_string.h"
 #include <stdbool.h>
 #include <math.h>
+#include <float.h>
+
+#define MATH_PI  3.14159265358979323846f
+#define MATH_HALF_PI 1.57079632679489661923f
+#define MATH_TWO_PI 6.28318530717958647692f
+#define DEG_TO_RAD 0.017453292519943295769236907684886f
+#define RAD_TO_DEG 57.295779513082320876798154814105f
```

Comment:
Is this necessary?

Reply:
It's used be the newly added API, following the original code style.